frappe/model/meta.py

```diff
@@ -530,6 +530,18 @@ def get_high_permlevel_fields(self):
 
 		return self.high_permlevel_fields
 
+	def get_permlevel_read_fields(self, parenttype=None, *, user=None):
```

Comment:
Isn't this basically `get_readable_fields`? `get_permlevel_read_fields` makes it sound like it's permlevel-wise dict of fields.

I would rename the variable below as well.

Reply:
fields suggests DocFields will be returned (seeing what other methods in the class suggest). Changing this to `get_permitted_fieldnames` with consideration with the suggestions below.